lib/shopify_cli/identity_auth.rb

```diff
@@ -255,15 +255,15 @@ def challange_params
     def auth_url
       if Environment.use_local_partners_instance?
         "https://identity.myshopify.io/oauth"
-      elsif Environment.use_spin_partners_instance?
+      elsif Environment.use_spin?
         "https://identity.#{Environment.spin_url}/oauth"
```

Comment:
Already the `SPIN_PARTNERS` variable was being used to mean that spin was being used for all services as shown here in the identity auth module.